beast/observationmodel/ast/make_ast_xy_list.py

```diff
@@ -20,12 +20,11 @@ def pick_positions_from_map(
     N_bins,
     bin_width,
     custom_bins,
-    Npermodel,
+    Nrealize,
```

Comment:
Why change this from a keyword to a passed argument?  This may break other peoples code.   No idea, but would like to hear more why this change.

Reply:
This is actually not a change. It's a cleanup and, in fact, a fix. The function takes both Nrealize and Npermodel, which is the same thing, and their usage is mixed up here and there. To remove confusion, I had to pick one and clean up the other. Do you prefer a keyword over a passed argument? Switching from a passed argument to a keyword is not difficult. This change would not break other people's code b/c (1) no one is repeating at the moment; (2) and if someone has been trying to repeat ASTs, they must have an issue that ASTs are not actually repeated.

Reply:
Ah, that makes sense.  Great.  Thanks for doing this!  I am find with your changes.